packages/assets-controllers/CHANGELOG.md

```diff
@@ -1903,7 +1912,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Use Ethers for AssetsContractController ([#845](https://github.com/MetaMask/core/pull/845))
```

Comment:
What is the effect of this change? Does this change the implementation or do we generally have the same behavior? Curious if we should mention this at all or if it's a developer-only thing.

``` suggestion
```

Reply:
Ah you are right. Sorry 😬